target/linux/ramips/dts/HC5861.dts

```diff
@@ -90,30 +84,29 @@
 	mdio-bus {
 		status = "okay";
 
-		phy4: ethernet-phy@4 {
-			reg = <4>;
-			phy-mode = "rgmii";
-		};
-
 		phy5: ethernet-phy@5 {
 			reg = <5>;
 			phy-mode = "rgmii";
 		};
 	};
 };
 
-&gsw {
-	mediatek,port4 = "gmac";
-};
-
 &pcie {
 	status = "okay";
 };
 
 &pcie0 {
-	mt76@0,0 {
+	wifi@0,0 {
+		compatible = "pci14c3,7662";
 		reg = <0x0000 0 0 0 0>;
 		mediatek,mtd-eeprom = <&factory 0x8000>;
 		ieee80211-freq-limit = <5000000 6000000>;
 	};
 };
+
+&state_default {
+	gpio {
+		ralink,group = "uartf", "wled", "rgmii2", "ephy";
```

Comment:
It seems that this comment is too late as this got merged already. :(
This router only used uartf pins so you only need uartf here. There is no need to set unused pinmux groups in dts.

Reply:
And uartf is already added in HC5X61.dtsi so this pinmux node can be dropped.

Reply:
I don't know what it means, I thought that if these are not used for pinctrl, they should be put in the gpio group.

Anyway, it seems that there is no problem?

Reply:
@ptpt52 I might not able to explain it clear enough in English so I've sent some Chinese explanation to your mail :)